Testscripts/Linux/Sysbench.sh

```diff
@@ -0,0 +1,210 @@
+    sysbench cpu --num-threads=1 run > /home/lisa/cpu.log
+    if [ $? -ne 0 ]; then
+        LogMsg "ERROR: Unable to execute sysbench CPU. Aborting..."
+        SetTestStateAborted
+    fi
+
+    PASS_VALUE_CPU=`cat /home/lisa/cpu.log |awk '/total time: / {print $3;}'`
+    if [ $? -ne 0 ]; then
+        LogMsg "ERROR: Cannot find cpu.log."
+        SetTestStateAborted
+    fi
+
+    RESULT_VALUE=$(echo ${PASS_VALUE_CPU} | head -c2)
+    if  [ $RESULT_VALUE -lt 15 ]; then
+        CPU_PASS=0
+        LogMsg "CPU Test passed. "
+    fi
+    LogMsg "`cat /home/lisa/cpu.log`"
+    return "$CPU_PASS"
+}
+
+function File-IO ()
+{
+    sysbench fileio --num-threads=1 --file-test-mode=$1 prepare > /dev/null 2>&1
+    LogMsg "Preparing files to test $1..."
+    sysbench fileio --num-threads=1 --file-test-mode=$1 run > /home/lisa/$1.log
+    if [ $? -ne 0 ]; then
+        LogMsg "ERROR: Unable to execute sysbench fileio mode $1. Aborting..."
+        SetTestStateFailed
+    else
+        LogMsg "Running $1 tests..."
+    fi
+
+    PASS_VALUE_FILEIO=`cat /home/lisa/$1.log |awk '/sum/ {print $2;}' | cut -d. -f1`
+    if [ $? -ne 0 ]; then
+        LogMsg "ERROR: Cannot find $1.log."
+        SetTestStateFailed
+    fi
+
+    if  [ $PASS_VALUE_FILEIO -lt 12000 ]; then
+        FILEIO_PASS=0
+        LogMsg "Fileio Test -$1- passed with latency sum: $PASS_VALUE_FILEIO."
+    else
+        LogErr "ERROR: Latency sum value is $PASS_VALUE_FILEIO. Test failed."
+    fi
+
+    sysbench fileio --num-threads=1 --file-test-mode=$1 cleanup
+    LogMsg "Cleaning up $1 test files."
+
+    LogMsg "`cat /home/lisa/$1.log`"
+    cat /home/lisa/$1.log >> /home/lisa/fileio.log
+    rm /home/lisa/$1.log
+    return "$FILEIO_PASS"
+}
+#######################################################################
+#
+# Main script body
+#
+#######################################################################
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    exit 0
+}
+
+#
+# Source constants file and initialize most common variables
+#
+UtilsInit
+
+LogMsg "Updating test case state to running"
+
+if [ -e ./${CONSTANTS_FILE} ]; then
+    source ${CONSTANTS_FILE}
+else
+    LogMsg "Error: no ${CONSTANTS_FILE} file"
+    SetTestStateAborted
+    exit 10
+fi
+
+# Download sysbench
+pushd $ROOT_DIR
+LogMsg "Cloning sysbench"
+wget https://github.com/akopytov/sysbench/archive/$Sysbench_Version.zip
+if [ $? -gt 0 ]; then
+    LogMsg "Failed to download sysbench."
+    SetTestStateFailed
+    exit 10
+fi
+
+unzip $Sysbench_Version.zip
+if [ $? -gt 0 ]; then
+    LogMsg "Failed to unzip sysbench."
+    SetTestStateFailed
+    exit 10
+fi
+
+if is_fedora ; then
+    pushd $ROOT_DIR
+    mkdir autoconf
+    cd autoconf
+    wget http://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.gz
+    tar xvfvz autoconf-2.69.tar.gz
+    cd autoconf-2.69
+    ./configure
+    make
+    make install
+
+    yum install devtoolset-2-binutils -y
```

Comment:
there should be a function to install the packages

Reply:
Created function for dependencies and sysbench install